org.lflang/src/org/lflang/ast/ToLf.java

```diff
@@ -757,14 +760,26 @@ public MalleableString caseSerializer(Serializer object) {
   @Override
   public MalleableString caseKeyValuePairs(KeyValuePairs object) {
     // {KeyValuePairs} '{' (pairs+=KeyValuePair (',' (pairs+=KeyValuePair))* ','?)? '}'
-    if (object.getPairs().isEmpty()) return MalleableString.anyOf("");
+    if (object.getPairs().isEmpty()) {
+      return MalleableString.anyOf("");
+    }
     return new Builder()
         .append("{\n")
         .append(list(",\n", "", "\n", true, true, object.getPairs()).indent())
         .append("}")
         .get();
   }
 
+  @Override
+  public MalleableString caseBracedListExpression(BracedListExpression object) {
+    if (object.getItems().isEmpty()) {
+      return MalleableString.anyOf("{}");
+    }
+    // Note that this strips the trailing comma. There is no way
+    // to implement trailing commas with the current set of list() methods AFAIU.
```

Comment:
This not a problem for brace lists in C/C++, but it might become a problem for tuples in Python. That is a problem for another day though...

Reply:
That's true. I'm not sure it makes sense to add a dedicated syntax for Python tuples anyway... It sounds like it would be rarely used.